src/devrules/core/project_service.py

```
@@ -451,6 +451,59 @@ def add_issue_comment(owner: str, repo: str, issue: int, comment: str) -> None:
+        "item-list",
+        project_number,
+        "--owner",
+        owner,
+        "--format",
+        "json",
+        "--limit",
+        "100",
+    ]
+
+    try:
+        result = subprocess.run(
```

Ruff (0.14.8)

`subprocess` call: check for execution of untrusted input

(S603)